# Obey CMAKE_INSTALL_PREFIX
# July 10th 2018 by B. Hajgato (Free University Brussels - VUB)
--- cmake/custom/own.cmake.orig	2018-04-20 09:44:47.000000000 +0200
+++ cmake/custom/own.cmake	2018-07-10 16:08:05.999455805 +0200
@@ -1,4 +1,4 @@
-set(CMAKE_INSTALL_PREFIX ${PROJECT_BINARY_DIR})
+#set(CMAKE_INSTALL_PREFIX ${PROJECT_BINARY_DIR})
 
 include(GNUInstallDirs)
 
